Update automation troubleshooting

zairhenrique opened the pull request:
Automation accounts now doesnt have an option to select tier. From now, you must enable paid tier to use Basic features.

I think its reasonable to update docs for the new format.

Regards,

azurecla commented:
Hi __@ClarityQueen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

zairhenrique committed:
Update automation troubleshooting 

Automation accounts now doesnt have an option to select tier. From now, you must enable paid tier to use Basic features.

zairhenrique committed:
Merge branch 'master' into patch-1
